src/elarm_server.erl

```diff
@@ -549,6 +558,15 @@ handle_get_alarms(#state{ alarmlist_cb = AlCB,
     {Result, NewAlState} = AlCB:get_alarms(AlState),
     {Result, State#state{ alarmlist_state = NewAlState}}.
 
+handle_get_alarm(AlarmId, Src,
+                 #state{ alarmlist_cb = AlCB, alarmlist_state = AlState } = State) ->
```

Comment:
Line 562 is too long:                  #state{ alarmlist_cb = AlCB, alarmlist_state = AlState } = State) ->.